src/pages/routePropTypes.js

```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+    // The accountID and validateCode are passed via the URL
+    route: PropTypes.shape({
+        // Each parameter passed via the URL
+        params: PropTypes.shape({
+        }),
```

Comment:
Missing some information in this shape?

Reply:
Ah I left this intentionally vague so it could be used by other components, but you're right that we should specify that the params are accountID and validateCode somewhere.